config/helm/chart/default/templates/Common/webhook/deployment-webhook.yaml

```diff
@@ -21,7 +21,11 @@ metadata:
   labels:
     {{- include "dynatrace-operator.webhookLabels" . | nindent 4 }}
 spec:
+  {{- if (.Values.webhook).enableHighAvailability }}
```

Comment:
I think the `ternary` operator could improve readability here: [helm docs](https://helm.sh/docs/chart_template_guide/function_list/#ternary)